### DISCUSSION TITLE
cocos2d.h file not found

### DISCUSSION THREAD

**Discussion by ghost** [axmolengine/axmol#632] [Q&A] [answered]
Hello all.. hope you all are doing well.. 
now i have created one c++(cpp) file in my ADXE project and added that file in "CMakeLists.txt" like below. i am attaching the screenshot for more understand the issue.. after all the necessary steps i have got cocos2d.h file not found in new created cpp file only. so kindly need to understand what mistake i did and how i can go further... 
![Screenshot 2022-05-10 at 9 44 31 PM](https://user-images.githubusercontent.com/105281914/167675073-9214d752-569f-4098-817c-37b23420c5ff.png)

![Screenshot 2022-05-10 at 9 43 01 PM](https://user-images.githubusercontent.com/105281914/167675097-19d35da2-c422-4062-92d7-8d64ef3db252.png)

**Comment by DelinWorks** [comment, marked as answer]
You included cocos2d in the hpp file right? why would you include the Hello.hpp in it self again?

**Reply by ghost** [reply]
@DelinWorks thanks for your replay. but i used this solution and it's worked for me..
https://stackoverflow.com/questions/9799130/import-cocos2d-h-works-in-some-files-but-not-all

again thank you @DelinWorks..
